src/cargo/util/lints.rs

```diff
@@ -249,10 +405,6 @@ pub fn check_im_a_teapot(
     let manifest = pkg.manifest();
     let (lint_level, reason) = IM_A_TEAPOT.level(pkg_lints, ws_lints, manifest.edition());
 
-    if IM_A_TEAPOT.is_feature_gated(manifest) {
-        return Ok(());
-    }
```

Comment:
If I'm understanding this, unstable lints run, we just also error.  Is that right?

Should we have the "allow" logic also force these as "allow", causing them to be bypassed automatically for every unstable lint?